srcpkgs/alpine/template

```diff
@@ -29,7 +29,3 @@ do_build() {
 		CC="$BUILD_CC" CFLAGS="$BUILD_CFLAGS" CPPFLAGS= LDFLAGS="$BUILD_LDFLAGS"
 	make ${makejobs} all CC="$CC"
 }
-
-post_install() {
-	vlicense NOTICE
-}
```

Comment:
why are you removing this?

Reply:
xlint complained that license Apache-2.0 should not be installed. I don't know if there's another solution

Reply:
NOTICE suggests license should probably be `Apache-2.0, BSD-4-Clause`

Reply:
Okay, will try that